borg/remote.py

```diff
@@ -176,16 +175,11 @@ def __init__(self, location, create=False, lock_wait=None, lock=True, append_onl
                 raise ConnectionClosedWithHint('Is borg working on the server?') from None
             if version != RPC_PROTOCOL_VERSION:
                 raise Exception('Server insisted on using unsupported protocol version %d' % version)
-            # Because of protocol versions, only send append_only if necessary
-            if append_only:
-                try:
-                    self.id = self.call('open', self.location.path, create, lock_wait, lock, append_only)
-                except self.RPCError as err:
-                    if err.remote_type == 'TypeError':
-                        raise self.NoAppendOnlyOnServer() from err
```

Comment:
Why remove this? Silently ignoring Repository settings on create doesn‘t sound nice.